src/os/ui/areas.js

```diff
@@ -144,13 +147,25 @@ os.ui.AreasCtrl.prototype.exportDisabled = function() {
 };
 
 
+/**
+ * Default for the export button
+ *
+ * @export
+ */
+os.ui.AreasCtrl.prototype.defaultExport = function() {
+  os.ui.AreasCtrl.export({areaOpt: 'selected', scope: this.scope});
+};
+
+
 /**
  * Pop up area export gui
  *
+ * @param {Object} opt
```

Comment:
In a similar vein to the above comment, we generally prefer to define a `typedef` rather than using generic options objects, just to make it clear what the type of each value is on here. If you look at `importmanager.js:16`, there's an example declaration of `os.ui.im.ImporterConfig`. All this does is tell the Closure compiler what type those values are.